spec/lib/authorities/linked_data/term_config_spec.rb

```diff
@@ -156,7 +172,7 @@
       expect(min_config.term_language).to eq nil
     end
     it 'returns the preferred language for selecting literal values if configured for term' do
-      expect(full_config.term_language).to eq [:en]
+      expect(full_config.term_language).to eq [:es]
```

Comment:
This value was changed in `lod_full_config.json` fixture to make each language different so we can be sure the language we are checking came for the right place in the config.